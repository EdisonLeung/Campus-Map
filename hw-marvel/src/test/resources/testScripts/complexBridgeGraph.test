# Tests pathfinding on a complex graph where there is one node "bridge" that connect
# one group of nodes to another

# Load Graph
LoadGraph bridge bridgeGraph.csv

# pathfinding test
FindPath bridge a b

FindPath bridge f d

FindPath bridge b d

FindPath bridge a f

FindPath bridge e c

FindPath bridge c d