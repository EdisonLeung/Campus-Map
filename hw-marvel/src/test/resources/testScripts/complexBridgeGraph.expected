# Tests pathfinding on a complex graph where there is one node "bridge" that connect
# one group of nodes to another

# Load Graph
loaded graph bridge

# pathfinding test
path from a to b:
a to b via AB

path from f to d:
f to e via EF
e to d via DE

path from b to d:
b to d via BD

path from a to f:
a to b via AB
b to d via BD
d to e via DE
e to f via EF

path from e to c:
e to d via DE
d to b via BD
b to c via BC

path from c to d:
c to b via BC
b to d via BD